packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -1019,7 +1019,10 @@ class FlutterVmService {
         onError: (Object? error, StackTrace stackTrace) {
           if (error is vm_service.SentinelException ||
             error == null ||
-            (error is vm_service.RPCError && error.code == RPCErrorCodes.kServiceDisappeared)) {
+            error is vm_service.RPCError &&
```

Comment:
@bkonyi chrome proxy service throws a sentinel exception on `getIsolate` for non-existing isolate id, as required by the VmServiceInterface, but the VmServerConnection translates it into an RPC error with RPCError.kInternalError as code, and the sentinel is added to the message as a string, so I have to check for the string contents here. Is there a better way to handle this error? Should we change the VMServer connection to throw something different, or have a set of error codes in `package:vm_service` we all agree on and can check for here?